tests/unit/schema/test_column_schemas.py

```diff
@@ -13,15 +13,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-import numpy
 import pytest
 
+from merlin import dtype
 from merlin.schema import ColumnSchema
 from merlin.schema.schema import ColumnQuantity
 from merlin.schema.tags import Tags, TagSet
 
 
-@pytest.mark.parametrize("d_types", [numpy.float32, numpy.float64, numpy.uint32, numpy.uint64])
+@pytest.mark.parametrize("d_types", [dtype.float32, dtype.float64, dtype.uint32, dtype.uint64])
```

Comment:
These dtypes should get normalized to Merlin types anyway when creating a `ColumnSchema`, so I'm not sure if this test actually needs to change. Should check that.

Reply:
The conversion still works fine with `numpy.whatever` types but that makes it harder to write the test assertions, so it actually does make sense to convert these over (just for convenience.)